spago.dhall

```diff
@@ -0,0 +1,26 @@
+{ name = "elmish"
+, description = "A PureScript implementation of The Elm Architecture (TEA)"
+, authors =
+    [ "Fyodor Soikin <fyodor@collegevine.com>"
+    , "CollegeVine"
+    ]
```

Comment:
These fields aren't technically part of `spago.dhall` schema - it looks like Spago doesn't support description and authors at all. But I put them here anyway, so they're not lost. Spago just ignores them.